specs/_features/focil/p2p-interface.md

```diff
@@ -0,0 +1,48 @@
+# FOCIL -- Networking
+
+This document contains the consensus-layer networking specification for FOCIL.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Time parameters](#time-parameters)
+- [The gossip domain: gossipsub](#the-gossip-domain-gossipsub)
+  - [Topics and messages](#topics-and-messages)
+    - [Global topics](#global-topics)
+      - [`local_inclusion_list`](#local_inclusion_list)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+### Time parameters
+
+| Name | Value | Unit | Duration |
+| - | - | :-: | :-: |
+| `LOCAL_INCLUSION_LIST_CUT_OFF` | `uint64(9)` | seconds | 9 seconds |
+
+
+### The gossip domain: gossipsub
+
+#### Topics and messages
+
+The new topics along with the type of the `data` field of a gossipsub message are given in this table:
+
+| Name                          | Message Type                                         |
+|-------------------------------|------------------------------------------------------|
+| `local_inclusion_list`    | `SignedLocalInclusionList` [New in FOCIL] |
+
+##### Global topics
+
+FOCIL introduces new global topics for inclusion list and inclusion list aggregate.
+
+###### `local_inclusion_list`
+
+This topic is used to propagate signed local inclusion list as `SignedLocalInclusionList`.
+The following validations MUST pass before forwarding the `local_inclusion_list` on the network, assuming the alias `message = signed_local_inclusion_list.message`:
+
+- _[REJECT]_ The slot `message.slot` is equal to current slot.
+- _[IGNORE]_ The current time is `LOCAL_INCLUSION_CUT_OFF` seconds into the slot.
+- _[REJECT]_ The transactions `message.transactions` length is within upperbound `MAX_TRANSACTIONS_PER_INCLUSION_LIST`.
+- _[IGNORE]_ The `message` is the first valid message received from the validator with index `message.validate_index` and can be received no more than twice from the same validator index.
```

Comment:
Does the IL equivocation handling mechanism need to be specified somewhere?

Reply:
Ya I need to add that to the fork choice spec...